activesupport/lib/active_support/descendants_tracker.rb

```diff
@@ -86,10 +86,18 @@ def subclasses
         subclasses
       end
 
-      def descendants
-        descendants = super
-        descendants.reject! { |d| @@excluded_descendants[d] }
-        descendants
+      if RubyFeatures::CLASS_DESCENDANTS
```

Comment:
I am not sure if a future version of Ruby which implements `Class#descendants` behaves as written in this code. Of course, we can fix it when `Class#descendants` is available but not sure if we need to implement it for now.

Reply:
Technically it was reverted and the ticket is now open. I don't think the spec is likely to change, from what I gathered the concern was about the internal implementation stability.

Also it's not just MRI. Truffle is likely to implement both right away. So I think it's fine to be a bit future proof here.

Reply:
Understood and agreed.